benchmarks-jmh/build.gradle

```diff
@@ -2,12 +2,15 @@ apply plugin:'application'
 
 mainClassName = 'org.openjdk.jmh.Main'
 
+sourceCompatibility = 1.8
```

Comment:
Is 1.8 language level really necessary for these benchmarks? If it is, we'll accept it. But we are trying to keep our project on 1.7 by now.